src/index.js

```diff
@@ -61,13 +61,14 @@ function rollem (configs, options) {
       if (err) {
         watcher.emit('failed', err)
       } else {
-        gazeWatcher.on('all', () => {
-          watcher.emit('changed')
-          buildBundles(configs).then(() => watcher.emit('rolled'))
+        gazeWatcher.on('all', (_event, filepath) => {
```

Comment:
I checked your link and tested out this example code on the demo page, which is similar to your pattern:
```javascript
/* global document */

document.addEventListener('click', function(e, filepath){
  document.body.textContent = filepath
})
```
ESLint responded with everything being okay.